x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/stats.tsx

```diff
@@ -110,10 +110,12 @@ export function MobileStats({
         defaultMessage: 'Crash rate',
       }),
       icon: getIcon('bug'),
-      value: data?.currentPeriod?.crashRate?.value ?? NOT_AVAILABLE_LABEL,
+      value: data?.currentPeriod?.crashRate?.value ?? NaN,
```

Comment:
This change looks wrong. I don't think we want to default to `NaN`

Reply:
After looking at the code I see why you had to do this. It still pains my hard that we have to make changes with runtime impact only to satisfy a compile-time check. It might be better to adjust the types of `MetricDatum` but that's a little out of the scope of this change.

Reply:
Hey @sqren, I was going to explain this, `NaN` was the previous fallback value before the `NOT_AVAILABLE_LABEL` constant was introduced, which was actually the change breaking the typing of the `valueFormatter` option. I agree the change should occur at the typing level, but as you said it would require a bigger change, I preferred to revert this to its previous implementation and handle it in the `valueFormatter` implementation to scope the impact to these lines only.